frontend/public/components/conditions.tsx

```diff
@@ -2,9 +2,20 @@ import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import { LinkifyExternal, Timestamp } from './utils';
 import { CamelCaseWrap } from './utils/camel-case-wrap';
-import { K8sResourceCondition } from '../module/k8s';
+import { ClusterServiceVersionCondition, K8sResourceCondition } from '../module/k8s';
 
-export const Conditions: React.FC<ConditionsProps> = ({ conditions }) => {
+/**
+ * Since ClusterServiceVersionCondition type is different from K8sResource, but InstallPlanCondition and SubscriptionCondition are identical, we will use the following enum to render the proper conditions table based on type.
```

Comment:
```suggestion
 * Since ClusterServiceVersionCondition type is different from K8sResourceCondition, but InstallPlanCondition and SubscriptionCondition are identical, we will use the following enum to render the proper conditions table based on type.
```

Reply:
@jhadvig done